olm/openebsoperator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,424 @@
+              "imageTag": "1.0.0",
+              "generateTLS": "true",
+              "replicas": "1"
+            },
+            "jiva": {
+              "image": "quay.io/openebs/jiva",
+              "imageTag": "1.0.0",
+              "replicas": "3"
+            },
+            "cstor": {
+              "pool": {
+                "image": "quay.io/openebs/cstor-pool",
+                "imageTag": "1.0.0"
+              },
+              "poolMgmt": {
+                "image": "quay.io/openebs/cstor-pool-mgmt",
+                "imageTag": "1.0.0"
+              },
+              "target": {
+                "image": "quay.io/openebs/cstor-istgt",
+                "imageTag": "1.0.0"
+              },
+              "volumeMgmt": {
+                "image": "quay.io/openebs/cstor-volume-mgmt",
+                "imageTag": "1.0.0"
+              }
+            },
+            "policies": {
+              "monitoring": {
+                "enabled": "true",
+                "image": "quay.io/openebs/m-exporter",
+                "imageTag": "1.0.0"
+              }
+            },
+            "analytics": {
+              "enabled": "true",
+              "pingInterval": "24h"
+            }
+          }
+        }
+      ]
+    description: Creates and maintains OpenEBS Control Plane deployments
+spec:
+  displayName: OpenEBS
+  description: >
+    **OpenEBS** is a container attached storage solution that enables the use 
+    of containers for mission-critical, persistent workloads and for other stateful 
+    workloads such as logging or Prometheus for example.
+
+    OpenEBS itself is deployed as just another container on your host and enables 
+    storage services that can be designated on a per pod, application, cluster 
+    or container level, including:-
+
+    * Data persistence across nodes
+    * Synchronization of data across availability zones and cloud providers
+    * A common layer so whether you are running on the cloud, or your bare metal
+    * Integration with Kubernetes, so developer and application intent flows into OpenEBS
+    * Management of tiering to and from S3 and other targets.
+
+    # Start using OpenEBS
+    
+    * Try the quickstart [guide](https://docs.openebs.io/docs/next/quickstart.html)
+    
+    # Contribute to OpenEBS
+
+    * To learn how to contribute, please read the contribution guide: 
+      [contribution guide](https://github.com/openebs/openebs/blob/master/CONTRIBUTING.md)
+
+    # Community
+
+    * OpenEBS welcomes your feedback and contributions in any form possible.
+      [Join our Community](https://openebs.io/join-our-slack-community)
+
+    # License
+ 
+    OpenEBS is distributed under the 
+    [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.txt)
+    
+  keywords:
+    - OpenEBS
+    - ContainerAttachedStorage
+  version: 1.0.0
+  maturity: alpha
+  maintainers:
+    - name: kmova
+      email: kiran.mova@openebs.io
```

Comment:
Include - all the maintainers karthik that we proposed in CNCF or .. is this about maintainers of helm operator?

Reply:
This is for maintainer of operator @kmova . Ref from OLM repo: 

```
Maintainers: A list of names and email addresses of the maintainers of the Operator code. This can be a list of individuals or a shared email alias, eg. support@example.com.
```